ui/src/app/pages/admin/institution/admin-institution-edit.spec.tsx

```diff
@@ -760,4 +793,21 @@ describe('AdminInstitutionEditSpec - add mode', () => {
     ).not.toBeInTheDocument();
     await user.unhover(getAddButton());
   });
+
+  it('Should allow setting bypassInitialCreditsExpiration', async () => {
```

Comment:
Is this different than 'Should allow updating bypassInitialCreditsExpiration from false to true'?

Reply:
this is the "add new Inst" case and the other is "edit existing Inst"